modules/gitgraph/graph_test.go

```diff
@@ -269,446 +269,446 @@ func TestCommitStringParsing(t *testing.T) {
 	}
 }
 
-var testglyphs = `* 
```

Comment:
Is this change intended?

Reply:
Argh! I always make the same mistake with that file.
The problem is, I ensured I replaced everything using
```sh
find . -type f -name '*.go' -print -exec vim {} -c ':%s/$OLD_FUNCTION/$NEW_FUNCTION/g' -c ':wq' \;
```
The problem is, my vim is configured to throw out trailing whitespace.
Will revert it now.

Reply:
vscode F2 is generally a safer way to refactor function names (unless of course they're called dynamically)